src/DGmethods/DGBalanceLawDiscretizations.jl

```diff
@@ -659,6 +688,64 @@ function grad_auxiliary_state!(disc::DGBalanceLaw, id, (idx, idy, idz))
                            Dmat, topology.elems, id, idx, idy, idz))
 end
 
+
+"""
+    indefinite_stack_integral!(disc, f, Q, out_states, [P=disc.auxstate])
```

Comment:
should be added to the relevant `docs/src/` file.

Reply:
done (as well as fixed other docs issues)